### PR TITLE
Prepare 1.26.1 release.

### PR DESCRIPTION
**Pull request by nc6**
This should not be merged until #2540 has been approved and merged.
Update changelogs and versions for 1.26.1 release.

### CODE REVIEW

**Comment by frog357** [comment]
There is a minor typo in the date: March 2020.

**Review by karknu — DISMISSED** [review]
You're missing the version bump for `cardano-node-chairman`.

**Review by Jimbo4350 — APPROVED** [review]
LGTM besides the version bump for the chairman. Do we also need to version bump cardano-topology?

**Review by dcoutts — APPROVED** [review]
LGTM

**Comment by nc6** [comment]
bors merge

File: cardano-node/ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+  (#2511)
+- Support for GHC 8.6.5 has been dropped. (#2507)
+- Various internal improvements and refactorings. (#2505)
+- Disable the "uncoupled blocks" metric. This was shown in profiling to have an
+  unfortunately large overhead. This reverts the change introduced in #2321.
+  (#2510)
+- Update the iohk-monitoring framework to fix a file descriptor leak. (#2518)
+
+### ledger changes
+- Fix an unevalutated thunk error in reward computation. (#2183)
```
Comment (newhoggy) [inline cardano-node/ChangeLog.md:16]: Typo `unevalutated`

**Pull request merged**